### ISSUE TITLE
Aperture Plugins

### ISSUE THREAD

**Issue by AleOrsenigo** [cormiertyshawn895/Retroactive#49]
Good Afternoon,

I’m writing you only to ask you a question in the old aperture application I was used to have some plugins installed.

But right now i can not install the same plugins on the aperture installed with retroactive.

Is there any solution?

Best regards.

**Comment by cormiertyshawn895** [comment]
What are the names of your plugins?

**Comment by AleOrsenigo** [comment]
Hi,

The plugins are the luminar 4 plugins

Best regards

Sent from my iPhone

On 9 Feb 2020, at 22:25, Tyshawn Cormier <notifications@github.com> wrote:

﻿

What are the names of your plugins?

**Comment by AleOrsenigo** [comment]
Good Afternoon,

With the old version of Aperture on Mojave There was a folder under Application Support where to insert them

Best Regards

**Comment by AleOrsenigo** [comment]
Hi, do you have any news regarding this problem with plugins?

Best Regards.

**Comment by cormiertyshawn895** [comment]
> Hi, do you have any news regarding this problem with plugins?

1. Download the Luminar 4 Plugin for Aperture.
[Luminar4Plugin.ApertureEdit.zip](https://github.com/cormiertyshawn895/Retroactive/files/4271112/Luminar4Plugin.ApertureEdit.zip)
1. Extract the plguin by double clicking on the downloaded zip
1. In Finder's menu bar, choose Go -> Go to Folder...
1. Enter "/Library/Application Support"
1. If there isn't a folder named "Aperture", Create a new folder named "Aperture"
1. Double click to open the "Aperture" folder
1. If there isn't a folder named "Plug-Ins", Create a new folder named "Plug-Ins"
1. Double click to open the "Plug-Ins" folder
1. Drag the unzipped Luminar4Plugin.ApertureEdit plugin into the "Plug-Ins" folder
1. Quit Aperture and open it again.

Basically it's the same as before, except you have to make the "/Library/Application Support/Aperture/Plug-Ins" folder yourself.

**Comment by cormiertyshawn895** [comment]
Retroactive has been discontinued, and does not support macOS Sequoia or later. You should transition from Retroactive to a wide range of supported apps, many of which are built into macOS or free to download. [Learn how to transition from Retroactive to supported apps](https://github.com/cormiertyshawn895/Retroactive/blob/master/TRANSITION.md).
